helpers/balanceHelper.ts

```diff
@@ -288,8 +318,8 @@ export const resolveGas = async (options: ResolveGasArguments) => {
     }
 }
 
-export const formatAmount = (unformattedAmount: bigint | unknown, decimals: number) => {
-    return (Number(BigInt(unformattedAmount?.toString() || 0)) / Math.pow(10, decimals))
+export const formatAmount = (unformattedAmount: bigint | unknown, decimals: number | null | undefined) => {
+    return (Number(BigInt(unformattedAmount?.toString() || 0)) / Math.pow(10, decimals || 1))
```

Comment:
This looks dangerous. Maybe you should never call this method if you're not sure that you have the decimals?